### ISSUE TITLE
Integration Megaissue

### ISSUE THREAD

**Issue by moonheart08** [CyclopsMC/IntegratedDynamics#384]
#### Issue type:
- :heavy_plus_sign: Feature request

#### Why I'm making this
Having a issue for each *possible* integration to the mod would be tedious, and would create tons of new issues, so having a mega issue could be advisable.
As such, I'm doing so. I've compiled a midsize list of mods that could potentially get integration added.
Mods listed here that have a issue already will get their issue linked with their name.
If I am uncertain if a mod can be integrated, i'll leave a note about it.

#### Mods that could do with integration

- [ ] Botania
- [ ] Applied Energistics 2
- [ ] Psi (Should be possible, PsiAPI exists)
- [ ] Extreme Reactors
- [ ] Gendustry
- [ ] Storage Drawers
- [ ] IndustrialCraft 2 (Uncertain, can't find an API.)
- [ ] Actually Additions
- [ ] OpenComputers [Tracking Issue](https://github.com/CyclopsMC/IntegratedDynamics/issues/378)
- [ ] Extra Utilities 2 (Uncertain, can't find an API.)
- [ ] Forestry (bee reading)
- [ ] Thermal Expansion (machine reading)
- [ ] Advanced Generators
- [ ] Teckle
- [ ] Immersive Engineering (machine reading)
- [ ] RailCraft (once updated)
- [ ] Ender I/O (Once Updated)
- [ ] RFTools
- [ ] Signals

#### How to get a mod added or it's status changed
Just @mention me in the comments to this issue with what you want done.

**Comment by josephcsible** [comment]
Re Extreme Reactors: I'm extremely upset by ZeroNoRyouki/BigReactors#74, and until he changes his mind about that, I'm opposed to doing anything that would improve the Extreme Reactors experience. By the way, this is a great example of why GPL/LGPL are much better license choices than MIT (if Big Reactors had been GPL/LGPL originally, Extreme Reactors being closed-source would be a violation of it).

Re the rest of the mods, can you list exactly what you'd like to be added for each, like I did in CyclopsMC/IntegratedDynamics-Compat#6?

**Comment by rubensworks** [comment]
@moonheart08 Good initiative! I've added some mods I was thinking of.

We'll never be able to implement compat for all of them ourselves though, so any community contributions are welcome :-)

**Comment by moonheart08** [comment]
Yup. Shame i dont know how to use forge. and for @josephcsible, I agree.

**Comment by tommysundling** [comment]
I often use Applied Energistics 2 for my storage solution and would love to see integration with that similar to what you are doing with Refined Storage. Having the option to use AE2 for storage and do all of the cabling with Integrated Dynamics/Tunnels would be great.

Just my two cents :-)

**Comment by 7eggert** [comment]
add to list: Integrated Dynamics / tunnels / …
